Tests/Realm.Tests/Database/InstanceTests.cs

```diff
@@ -763,7 +763,7 @@ public void GetInstance_WhenDynamicAndDoesntExist_ReturnsEmptySchema()
         [TestCase("søren")]
         public void GetInstance_WhenPathContainsNonASCIICharacters_ShouldWork(string path)
         {
-            var folder = Path.Combine(InteropConfig.DefaultStorageFolder, path);
+            var folder = Path.Combine(InteropConfig.GetDefaultStorageFolder("Could not determine a writable folder to store the Realm file. When constructing the RealmConfiguration, provide an absolute optionalPath where writes are allowed."), path);
```

Comment:
Here we don't expect an error, so the message can be a lot shorter - something like "No error expected here".